redbot/core/bot.py

```diff
@@ -1401,19 +1401,26 @@ async def send_to_owners(self, content=None, **kwargs):
 
         This logs failing sends
         """
+        cog_name = get_cog_by_call(self, "send_to_owners")
+
         destinations = await self.get_owner_notification_destinations()
 
         async def wrapped_send(location, content=None, **kwargs):
             try:
                 await location.send(content, **kwargs)
             except Exception as _exc:
                 log.error(
-                    "I could not send an owner notification to %s (%s)",
+                    "I could not send an owner notification from cog '%s' to %s (%s)",
+                    cog_name,
                     location,
                     location.id,
                     exc_info=_exc,
                 )
 
+        content = "Message from {cog_name}\n{content}".format(
```

Comment:
Can you please add ``:`` after ``Message from {cog_name}`` ?
My poor soul hurts without them.